data-processing-lib/python/src/data_processing/transform/binary_transform.py

```diff
@@ -32,12 +32,12 @@ def __init__(self, config: dict[str, Any]):
         """
         self.config = config
 
-    def transform_binary(self, base_name: str, byte_array: bytes) -> tuple[list[tuple[bytes, str]], dict[str, Any]]:
+    def transform_binary(self, file_name: str, byte_array: bytes) -> tuple[list[tuple[bytes, str]], dict[str, Any]]:
```

Comment:
I would not make this a primary parameter and it should be optional.  Not every transform will get its data from a file and/or doesn't care about the source of the data.  Some might argue we should have used **kwargs here, but I'm ok if we put it at the end and default it to None.

Reply:
Its the framework that invokes this, not the developer

Reply:
and it should be optional.  can you please make the change.  Transforms will not ONLY be used by the table processor/orchestrator.

Reply:
and of course, this provides backwards compatibility.  Never a bad bonus.

Reply:
no, it will break the code